integration-libs/cdc/tsconfig.schematics.json

```diff
@@ -400,6 +400,9 @@
       "@spartacus/cdc/core": ["../../integration-libs/cdc/core/public_api"],
       "@spartacus/cdc": ["../../integration-libs/cdc/public_api"],
       "@spartacus/cdc/root": ["../../integration-libs/cdc/root/public_api"],
+      "@spartacus/cdc/manage-users": [
+        "../../integration-libs/cdc/manage-users/public_api"
+      ],
       "@spartacus/cdc/user-account": [
```

Comment:
please move this to the last of this array.

Reply:
done